zen-quotes-app/src/app/home/home.component.ts

```diff
@@ -0,0 +1,65 @@
+import {Component, OnInit} from '@angular/core';
+import {QuotesService} from "../quotes.service";
+import {Quote} from "../../types/quote.type";
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss']
+})
+
+export class HomeComponent implements OnInit {
+  title = "Inspiration";
+  quotes: Quote[] = [];
+  authors: string[] = [];
+  authorsFilter: string[] = [];
+  randomQuote: Quote = {author: "", text: ""};
+  quotesFiltered: Quote[] = [];
+  quotesPerPage: number = 9;
+  pagesArray: number[] = [];
+  activePage: number = 0;
+
+  constructor(private quotesService: QuotesService) {}
+
+  getAuthors() {
```

Comment:
By convention, any function that:
- starts with `get` would be expected to run a value.
- starts with `set` would be expected to take a value as a parameter and set a value.

Challenge: can you refactor the `getAuthors` function so that its name matches the convention and so that it accepts the authors as an input parameter?

Thanks! 🙏 🙇

Reply:
@jvandemo  could you help with an example and test here, when you have time, please 🙏🙇

Reply:
@IraLysenko Test suite was added in https://github.com/IraLysenko/zenQuotesApp/pull/3 with a challenge in the comments. ⚡ 😅 🙌 🙇

Reply:
@jvandemo Great! thank you very much! really appreciate your time and help! 😊🤗